mobile/lib/components/teraz_gramy/teraz_gramy_widget.dart

```diff
@@ -0,0 +1,53 @@
+import 'package:audio_service/audio_service.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:radioaktywne/components/ra_playbutton.dart';
+import 'package:radioaktywne/components/utility/color_shadowed_card.dart';
+import 'package:radioaktywne/components/utility/image_with_overlay.dart';
+import 'package:radioaktywne/extensions/build_context.dart';
+
+/// Widget representing what's currently played on the radio
+///
+/// Consists of a [ColorShadowedCard] with an image, icon ant text overlay.
+class TerazGramyWidget extends StatelessWidget {
+  const TerazGramyWidget({
+    super.key,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return ColorShadowedCard(
+      shadowColor: context.colors.highlightRed,
+      child: ImageWithOverlay(
+        isLoading: false,
```

Comment:
this illustrates my point from `lib/components/utility/image_with_overlay.dart`'s comments: it's kinda awkward to just say "isLoading: false".